### PR TITLE
powershell, pwsh, windows/*: fix PowerShell conventions

### PR DESCRIPTION
**Pull request by reinhart1010**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

This PR fixes the following PowerShell command documentation to match the current PowerShell conventions.

+ **Add-AppxPackage:** Use the original naming format (`Add-AppxPackage` instead of `add-appxpackage`), see similar naming concerns for [slmgr / slmgr.vbs](https://github.com/tldr-pages/tldr/pull/8201)
+ **Delete-Item:** Use commas for separating multiple files and directories, see discussion from https://github.com/tldr-pages/tldr/pull/11342#discussion_r1377320108
+ **powershell, pwsh:** Clarify the documentation that `powershell` refers to the original Windows version of PowerShell (version 5.1 and below), while `pwsh` refers to the PowerShell version 6 and above (formerly known as PowerShell Core, now known as "cross-platform PowerShell"); reverting changes on #7984 and #11264; see also https://stackoverflow.com/questions/60124810/what-is-the-difference-between-pwsh-and-powershell-integrated-console-on-vs#60132705
+ **Remove-AppxPackage:** Use the original naming format (`Remove-AppxPackage` instead of `remove-appxpackage`), see similar naming concerns for [slmgr / slmgr.vbs](https://github.com/tldr-pages/tldr/pull/8201)
+ **sls:** Fixed mistakes that this is an alias to `Where-Object`. The command is supposed to be an alias of `Select-String`.

### CODE REVIEW

File: pages/windows/remove-appxpackage.md [hunk 1]
```diff
@@ -1,20 +1,20 @@
-# remove-appxpackage
+# Remove-AppxPackage
 
 > A PowerShell utility to remove an app package from one or more user accounts.
-> More information: <https://learn.microsoft.com/powershell/module/appx/remove-appxpackage>.
+> More information: <https://learn.microsoft.com/powershell/module/appx/Remove-AppxPackage>.
```
Comment (kbdharun) [inline pages/windows/remove-appxpackage.md:4]: Should the link be capitalized too?

Comment (reinhart1010) [reply pages/windows/remove-appxpackage.md:4]: Still redirected to the non-capitalized version.

**Comment by reinhart1010** [comment]
I decided to tag this as a mass change, as I fixed `sls` referring to `Where-Object` instead of `Select-String`. Additionally, there is a correction on the usage of `powershell` and `pwsh`, whereas the earlier refers to PowerShell 5.1 and below (now known as legacy Windows PowerShell), while the latter refers to PowerShell 6 and above (formerly known as PowerShell Core).

**Review by kbdharun — APPROVED** [review]
LGTM, thanks for the fixes.

----

I am probably going to create a new release in the Node client today which will update the correct option to `-p` and `--platform` instead of `-o` and `--os `. So probably we can drop the sentence in this PR (as we no longer suggest C or the Elixr client in our README/Wiki) and the suggested ones will support the platform options.

**Comment by reinhart1010** [comment]
> So probably we can drop the sentence in this PR (as we no longer suggest C or the Elixr client in our README/Wiki)

How about waiting for another month or two until the update is rolled out to more people?

**Review by gutjuri — APPROVED** [review]

**Pull request merged**